.github/workflows/dart_ci.yaml

```diff
@@ -0,0 +1,33 @@
+name: Dart CI
```

Comment:
cribbed from other OSS dart repos at Workiva using [this search](https://sourcegraph.wk-dev.wdesk.org/search?q=file:dart_ci.yaml&patternType=literal)